sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneEventHubClient.cs

```diff
@@ -206,101 +205,103 @@ public override async Task<EventHubProperties> GetPropertiesAsync(CancellationTo
+        ///   Creates a consumer responsible for reading <see cref="EventData" /> from a specific Event Hub partition,
         ///   and as a member of a specific consumer group.
         ///
-        ///   A receiver may be exclusive, which asserts ownership over the partition for the consumer
-        ///   group to ensure that only one receiver from that group is reading the from the partition.
-        ///   These exclusive receivers are sometimes referred to as "Epoch Receivers."
+        ///   A consumer may be exclusive, which asserts ownership over the partition for the consumer
+        ///   group to ensure that only one consumer from that group is reading the from the partition.
+        ///   These exclusive consumers are sometimes referred to as "Epoch Consumers."
         ///
-        ///   A receiver may also be non-exclusive, allowing multiple receivers from the same consumer
-        ///   group to be actively reading events from the partition.  These non-exclusive receivers are
-        ///   sometimes referred to as "Non-epoch Receivers."
+        ///   A consumer may also be non-exclusive, allowing multiple consumers from the same consumer
+        ///   group to be actively reading events from the partition.  These non-exclusive consumers are
+        ///   sometimes referred to as "Non-epoch Consumers."
         ///
-        ///   Designating a receiver as exclusive may be specified in the <paramref name="receiverOptions" />.
-        ///   By default, receivers are created as non-exclusive.
+        ///   Designating a consumer as exclusive may be specified in the <paramref name="consumerOptions" />.
+        ///   By default, consumers are created as non-exclusive.
         /// </summary>
         ///
         /// <param name="partitionId">The identifier of the Event Hub partition from which events will be received.</param>
-        /// <param name="receiverOptions">The set of options to apply when creating the receiver.</param>
+        /// <param name="eventPosition">The position within the partition where the consumer should begin reading events.</param>
+        /// <param name="consumerOptions">The set of options to apply when creating the consumer.</param>
         ///
-        /// <returns>An event receiver configured in the requested manner.</returns>
+        /// <returns>An consumer configured in the requested manner.</returns>
         ///
-        public override EventReceiver CreateReceiver(string partitionId,
-                                                     EventReceiverOptions receiverOptions)
+        public override EventHubConsumer CreateConsumer(string partitionId,
+                                                        EventPosition eventPosition,
+                                                        EventHubConsumerOptions consumerOptions)
         {
             TrackOne.PartitionReceiver CreateReceiverFactory()
```

Comment:
These should continue to be `Receiver` nomenclature, as they refer to the track one types, which do not use the new nomenclature.